rewards/aws/boost.py

```diff
@@ -10,15 +10,30 @@
 console = Console()
 
 
-def upload_boosts(boost_data, chain: str):
+def upload_boosts(boost_data, chain):
+    upload_boosts_to_aws(
+        boost_data,
+        chain,
+        "badger-boosts"
+    )
+    
+
+def upload_proposed_boosts(boost_data, chain: str):
+    upload_boosts_to_aws(
+        boost_data,
+        chain,
+        "propose-boosts"
+    )
+
+def upload_boosts_to_aws(boost_data, chain: str, file_name):
     """Upload boosts file to aws bucket
 
     :param test:
     :param boost_data: calculated boost information
     """
     discord_url = get_discord_url(chain, BotType.Boost)
     chain_id = env_config.get_web3(chain).eth.chain_id
```

Comment:
We have a `CHAIN_IDS` Dict that stores the IDs from the string chain.  This could be `chain_id = CHAIN_IDS[chain]` rather than importing web3

Reply:
There are also other chain_id calls that we can get rid of, better to do this in another pr with some more cleanup